models/command.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+const Command = require('../config/command');
+
+const MODEL = {
+    id: Joi
+        .number().integer().positive()
+        .description('Identifier of this command')
+        .example(123345),
+
+    labels: Joi
+        .array()
+        .items(Joi.string())
+        .description('Labels for command')
+        .example(['stable', 'latest', 'beta']),
```

Comment:
Just being pedantic:
Better examples of labels would be 'docker', 'git', 'node'.

'stable', 'latest', 'beta' are tags.

Reply:
This examples follows the existing `template`'s for the avoidance of confusion.
https://github.com/screwdriver-cd/data-schema/blob/master/models/template.js#L18
Thanks.

Reply:
`template`'s `labels` is confusing, too. 😢

Reply:
Yeah. I think we ended up using `tag` and `labels` is just there but not really used right now.

Reply:
Do not see a need to add `label` now. If we really want it, can do it in a second pass.

Reply:
Okay. I remove `labels` key at Phase 1.

Reply:
done.